Tools/CMake/AMReXConfig.cmake.in

```diff
@@ -231,9 +233,17 @@ if (@AMReX_CUDA@)
     endif ()
 endif ()
 
+# CMake targets
 include( "${CMAKE_CURRENT_LIST_DIR}/AMReXTargets.cmake" )
 
-if (@CMAKE_VERSION@ VERSION_GREATER_EQUAL 3.20 AND  @AMReX_CUDA@ )
+# CMake targets aliases: last dimension built will be our legacy target
+list(LENGTH AMReX_SPACEDIM list_len)
+math(EXPR list_last "${list_len} - 1")
+list(GET AMReX_SPACEDIM ${list_last} AMReX_SPACEDIM_LAST)
+add_library(AMReX::amrex ALIAS AMReX::amrex_${AMReX_SPACEDIM_LAST}d)
```

Comment:
This file seems to be allowed to be included multiple times (seen as a comment in the auto-generated `AMReXTargets.cmake`, which defines the actual ND targets). Thus, we need an include guard for `add_library`:  #3324